### DISCUSSION TITLE
How the project is going?

### DISCUSSION THREAD

**Discussion by lucasthomaz97** [huxingyi/dust3d#105] [Q&A]
Today i knew dust3d project, when i entered here on github i saw there are not many updates on source code recently, on jeremy's youtube channel there's not an upload since 7 months, then i ask you: the project is still on work, did it paused?
i wish to have a most simplified 3d tool and i liked dust3d too much.
if there's something i could help i want to.

**Comment by huxingyi** [comment]
It's under heavy development although not been commited to this repository yet.
For example:
https://github.com/huxingyi/solidboolean
https://github.com/huxingyi/isotropicremesher
